model/project_event_test.go

```diff
@@ -158,6 +162,8 @@ func (s *ProjectEventSuite) TestModifyProjectEventRedactsAllVars() {
 	s.Equal(evergreen.RedactedAfterValue, eventData.After.Vars.Vars["added"], "newly-added var should be redacted")
 	s.Equal(evergreen.RedactedAfterValue, eventData.After.Vars.Vars["modified"], "modified var should be redacted")
 	s.Empty(eventData.After.Vars.Vars["unmodified"], "unmodified var should be present but empty to indicate it wasn't changed")
+	s.Equal(after.GitHubAppAuth.AppID, eventData.After.GitHubAppAuth.AppID)
+	s.Equal(evergreen.RedactedAfterValue, string(eventData.After.GitHubAppAuth.PrivateKey))
```

Comment:
Can we add a test to check that it also redacts the before value?

Reply:
This is already tested on L149.